asteroid/models/base_models.py

```diff
@@ -387,6 +387,23 @@ def get_model_args(self):
         return model_args
 
 
+@script_if_tracing
+def get_shape(tensor):
```

Comment:
Let's call that `jit_get_shape` and move it to `utils.torch_utils`, we might need it somewhere else and it's more discoverable there.